src/main/java/com/github/alina/sesion4_challenge/ChallengeSeventeen.java

```diff
@@ -0,0 +1,15 @@
+package com.github.alina.sesion4_challenge;
+
+public class ChallengeSeventeen {
+    public static void main(String[] args) {
+        boolean hasUsername = true;
+        boolean hasPassword =true;
```

Comment:
Please don't forget to format the code. We lack a space between `=` and `true`.

![image](https://github.com/alinaitschool/itschool/assets/120710343/4929fb7e-0c37-40d6-acd0-823f6a2e8666)